plugins/destination/mssql/queries/values.go

```diff
@@ -14,14 +14,13 @@ func GetRows(table arrow.Table) ([][]any, error) {
 	for n := 0; n < int(table.NumCols()); n++ {
 		col := table.Column(n)
 		row := 0
-		for _, chunk := range col.Data().Chunks() {
-			for i := 0; i < chunk.Len(); i++ {
-				rows[row][n], err = getColValue(chunk, row)
```

Comment:
This comment might not make sense, but why not `getColValue(chunk, i)` instead?

Reply:
Yeah, overwriting the row might not make sense either (even if it only runs a single iteration every time)

Reply:
Maybe `We only support records with arrays of length == 1, so we always extract the first value`